Algolia migration guide (#1623)
meilisearch/documentation: learn/what_is_meilisearch/comparison_to_alternatives.md

### Meilisearch vs Algolia

Meilisearch was inspired by Algolia's product and the algorithms behind it. We indeed studied most of the algorithms and data structures described in their blog posts in order to implement our own. Meilisearch is thus a new search engine based on the work of Algolia and recent research papers.

Meilisearch provides similar features and reaches the same level of relevance just as quickly as its competitor.

If you are a current Algolia user considering a switch to Meilisearch, you may be interested in our [migration guide](/learn/getting_started/algolia_migration.md).

#### Key similarities

Some of the most significant similarities between Algolia and Meilisearch are:

> - [Features](https://docs.meilisearch.com/learn/what_is_meilisearch/overview.html#features) such as search-as-you-type, typo tolerance, faceting, etc.

Reviewer: ```suggestion
- [Features](/learn/what_is_meilisearch/overview.md#features) such as search-as-you-type, typo tolerance, faceting, etc.
```

### Meilisearch vs Algolia

Meilisearch was inspired by Algolia's product and the algorithms behind it. We indeed studied most of the algorithms and data structures described in their blog posts in order to implement our own. Meilisearch is thus a new search engine based on the work of Algolia and recent research papers.

> Meilisearch provides similar features and reaches the same level of relevance just as quickly as its predecessor.

Reviewer 2: Maybe `competitors` ? Also `just as` seems a bit modest , why not `even better` ?

If you are a current Algolia user considering a switch to Meilisearch, you may be interested in our [migration guide](/learn/getting_started/algolia_migration.md).

#### Key similarities

Some of the most significant similarities between Algolia and Meilisearch are listed below:

- [Features](https://docs.meilisearch.com/learn/what_is_meilisearch/overview.html#features) such as search-as-you-type, typo tolerance, faceting, etc.
- Fast results targeting an instant search experience (answers < 50 milliseconds)
- Objects you index are schemaless
- Support all JSON data types
- Operate asynchronously

> - Provide a similar query response

Reviewer 2: Maybe listing the features as separate bullet points is also more impactful. The user might be more interested in these features than for example `operate asynchronously`. My bad if it is not the case!

#### Key differences

Contrary to Algolia, Meilisearch is open-source and written in Rust, a modern systems-level programming language. Rust provides speed, portability, and flexibility, which makes the deployment of our search engine inside Virtual Machines, containers, or even Lambda@Edge a seamless operation.

One of Algolia's major assets is the robust worldwide infrastructure that they offer to their customers.

> Meilisearch currently delivers a search engine and is not in a position to provide a competitive infrastructure yet, but we hope one day to offer a highly available cloud solution that is simpler to deploy and maintain than Algolia's.

Reviewer 2: ```suggestion
Meilisearch currently delivers a search engine and is not in a position to provide a competitive infrastructure yet, but providing a highly available cloud solution that is simpler to deploy and maintain than Algolia's is on its way.
```
